src/app/leverage/provider.tsx

```diff
@@ -239,6 +245,51 @@ export function LeverageProvider(props: { children: any }) {
     fetchStats()
   }, [baseToken])
 
+  useEffect(() => {
+    async function fetchCostOfCarry(
+      jsonRpcProvider: providers.JsonRpcProvider,
+    ) {
+      try {
+        const poolDataProviderContract = new UiPoolDataProvider({
+          uiPoolDataProviderAddress: AaveV3Ethereum.UI_POOL_DATA_PROVIDER,
+          provider: jsonRpcProvider,
+          chainId: ChainId.mainnet,
+        })
+        const reserves = await poolDataProviderContract.getReservesHumanized({
+          lendingPoolAddressProvider: AaveV3Ethereum.POOL_ADDRESSES_PROVIDER,
+        })
+
+        const formattedPoolReserves = formatReserves({
+          reserves: reserves.reservesData,
+          currentTimestamp: Math.floor(Date.now() / 1000),
+          marketReferenceCurrencyDecimals:
+            reserves.baseCurrencyData.marketReferenceCurrencyDecimals,
+          marketReferencePriceInUsd:
+            reserves.baseCurrencyData.marketReferenceCurrencyPriceInUsd,
+        })
+
+        const borrowedAsset = formattedPoolReserves.find(
+          (asset) =>
+            asset.symbol.toLowerCase() === outputToken.borrowedAssetSymbol,
```

Comment:
`borrowedAssetSymbol` is always lowercase?

Reply:
yeah that's a good point...it _should_ be specified that way, but it relies on the future editor knowing it should be lowercase, so I updated it to force it always